schema-engine/connectors/sql-schema-connector/src/introspection_pair/default.rs

```diff
@@ -0,0 +1,159 @@
+use either::Either;
```

Comment:
I think this file is in two places now because of a rebase

Reply:
The other one should be kept (src/introspection/introspection_pair/default.rs)

Reply:
Indeed, something weird happened here 🤔 
Nice catch

Reply:
Done